### PR TITLE
added tests; small fixes

### PR DESCRIPTION
**Pull request by erpeg**
- in test_SeqAnalysis.py  added class for testing Sequence Visualizer
- in __main__.py changed one of parameter while plotting example plots, so it's more representative
- in SeqVisualizer.py fixed issue with wrong labeling of y ax

### CODE REVIEW

**Comment by erpeg** [comment]
Improved values for test visualization - gravy isn't always the best indicator for scatter point sizes
commit f7c20b770e1d8634abe65c8b5e1a4daa0b088d54

File: Tests/test_SeqAnalysis.py [hunk 1]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+    def test_seqvisualizer_class_static_directory(self):
+        """
+        Function that tests if static directory is created properly while creating static plot
+
+        :return: returns nothing if class passes tests, else function returns text message
+        """
+        self.v.bar(parameter='sequence_length', out='bar_temp', show=False)
+        if_path_exists = os.path.exists(f"{self.test_root_dir}/Plots/static")
+        assert if_path_exists is True, \
+            "function test_seqvisualizer_class_static_directory() didn't pass the test"
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:421]: You are not testing `test_seqvisualizer_class_static_directory()`, you are testing your class.

File: Tests/test_SeqAnalysis.py [hunk 2]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+    def test_seqvisualizer_class_interactive_directory(self):
+        """
+        Function that tests if interactive directory is created properly while creating interactive plot
+
+        :return: returns nothing if class passes tests, else function returns text message
+        """
+        self.v.bar_interactive(parameter='amino_acid_comp', out='bar_interactive_temp', show=False)
+        if_path_exists = os.path.exists(f"{self.test_root_dir}/Plots/interactive")
+        assert if_path_exists is True, \
+            "function test_seqvisualizer_class_interactive_directory() didn't pass the test"
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:432]: As above

File: Tests/test_SeqAnalysis.py [hunk 3]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+        parameters
+
+        :return: returns nothing if class passes tests, else function returns text message
+        """
+        self.v.bar(parameter='sequence_length', out='bar_test', show=False)
+
+        result = filecmp.cmp(f"{self.test_root_dir}/Plots/static/bar_test.png",
+                             f"{self.test_root_dir}/proper_Plots/static/bar.png", shallow=False)
+        assert result is True, \
+            f"function test_seqvisualizer_class_static_bar() didn't pass the test on plot bar"
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:446]: Same here, information about test failure will be provided by `unittest` module. You should try to precise what went wrong. Fortunatelly, test's names are self-explanatory so it's easy to deduce.

File: Tests/test_SeqAnalysis.py [hunk 4]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+        parameters
+
+        :return: returns nothing if class passes tests, else function returns text message
+        """
+        self.v.box(parameter='extinction_coefficient', out='box_test', show=False)
+
+        result = filecmp.cmp(f"{self.test_root_dir}/Plots/static/box_test.png",
+                             f"{self.test_root_dir}/proper_Plots/static/box.png", shallow=False)
+        assert result is True, \
+            f"function test_seqvisualizer_class_static_box() didn't pass the test on plot box"
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:460]: As above

File: Tests/test_SeqAnalysis.py [hunk 5]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+
+        :return: returns nothing if class passes tests, else function returns text message
+        """
+        self.v.scatter(x_parameter='sequence_length', y_parameter='theoretical_pi', z_parameter='molecular_weight',
+                       size=(12, 7), out='scatter_test', show=False)
+
+        result = filecmp.cmp(f"{self.test_root_dir}/Plots/static/scatter_test.png",
+                             f"{self.test_root_dir}/proper_Plots/static/scatter.png", shallow=False)
+        assert result is True, \
+            f"function test_seqvisualizer_class_static_scatter() didn't pass the test on plot scatter"
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:475]: As above

File: Tests/test_SeqAnalysis.py [hunk 6]
```diff
@@ -310,6 +330,161 @@ def test_seqanalyzer_terminal_aa_counter(self):
...
+                             f"{self.test_root_dir}/proper_Plots/static/box.png", shallow=False)
+        assert result is True, \
+            f"function test_seqvisualizer_class_static_box() didn't pass the test on plot box"
+
+    def test_seqvisualizer_class_static_scatter(self):
+        """
+        Function checks if plot is created properly by comparing newly created plot with ready plot, both with same
+        parameters
+
+        :return: returns nothing if class passes tests, else function returns text message
```
Comment (moozeq) [inline Tests/test_SeqAnalysis.py:467]: I didn't notice it before when reviewing, but you don't need to write `:return:` in docstring for unittests. It's not a big issue, but it obscures tests. Single line will be fine:
`"""Function checks if plot is created properly by comparing newly created plot with ready plot, both with same parameters"""`

Comment (moozeq) [reply Tests/test_SeqAnalysis.py:467]: Same for all tests above

**Review by moozeq — COMMENTED** [review]
Everything's fine. Docstrings and results' messages may be improved.

**Pull request merged**